package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@slack/bolt",
-  "version": "4.4.0",
+  "version": "4.4.0-aiAppsBeta.5",
```

suggestion: I think we want to revert this back
```suggestion
  "version": "4.4.0",
```

---

@mwbrooks Ahha perhaps I was too eager or still thinking of beta releases...